prelexer.cpp

```diff
@@ -434,6 +434,9 @@ namespace Sass {
           alternatives <
             alnum,
             exactly <'/'>,
+            exactly <'\\'>,
```

Comment:
Also, it looks like Ruby Sass treats the new line in the quoted string as a space delimited list.